tests/install.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- name: Install
+  hosts: all
+  pre_tasks:
+    - include: pre_tasks/manala.yml
+  vars:
```

Comment:
if the docker trick "--cap-add SYS_PTRACE" works, you could avoid setting configs in install test